File: java/divide.java
import java.util.Scanner;
public class Divide{
	public static void main(String[] args){
		int a, b;
		
		Scanner scan;		
		scan = new Scanner(System.in);
		
		System.out.println("Introduce el primer numero entero: ");
		a = scan.nextInt();
		System.out.println("Introduce el segundo numero entero: ");
		b = scan.nextInt();
		
		System.out.println("El resultado de la division es " + (a / b));
	}
}

File: c/divide.c
#include <stdio.h>
#include <stdlib.h>
int main(){
	int a, b;
	printf("Introduce el primer numero entero: ");
	scanf("%d", &a);
	printf("Introduce el segundo numero entero: ");
	scanf("%d", &b);
	printf("\nEl resultado de la division es %d", a / b);
	printf("\n");
	system("pause");
	return 0;
}